sdk/connect/javascript.md

```diff
@@ -39,23 +40,17 @@ import { MetaMaskSDK } from "@metamask/sdk"
 
 const MMSDK = new MetaMaskSDK({
   dappMetadata: {
-    name: "Example JavaScript Dapp",
+    name: "Example JavaScript dapp",
     url: window.location.href,
+    // iconUrl: "https://mydapp.com/icon.png" // Optional
   },
   infuraAPIKey: process.env.INFURA_API_KEY,
 })
+```
 
-const ethereum = MMSDK.getProvider()
-
-// Connect to MetaMask
-const accounts = await MMSDK.connect()
+- [`dappMetadata`](../reference/sdk-options.md#dappmetadata) - Ensures trust by showing your dapp's `name`, `url`, and `iconUrl` during connection.
 
-// Make requests
-const result = await ethereum.request({ 
-  method: "eth_accounts", 
-  params: [] 
-})
-```
+- [`infuraAPIKey`](../reference/sdk-options.md#infuraapikey) - Enables read-only RPC and load‑balancing.
```

Comment:
I mentioned in my previous comment, but can you move this list after the `</Tabs>` on line 75? And preface with something like:

> These examples configure the SDK with the following options: